<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AdvFind</class>
 <widget class="QDialog" name="Dialog_AdvFind">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowTitle" stdset="0">
   <string>Find in Files</string>
  </property>
  <property name="windowModality" stdset="0">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>290</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_top">
   <property name="topMargin">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_1">
     <item row="3" column="2">
      <widget class="QToolButton" name="folder_TB">
       <property name="text" stdset="0">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text" stdset="0">
        <string>File Type:</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="findType">
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="find">
       <property name="minimumSize" stdset="0">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text" stdset="0">
        <string>Find:</string>
       </property>
       <property name="minimumSize" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text" stdset="0">
        <string>Directory:</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="findFolder">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_1">
     <property name="sizeType" stdset="0">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>13</height>
      </size>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="sizeType" stdset="0">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="matchCase_CKB">
       <property name="text" stdset="0">
        <string>Match Case</string>
       </property>
       <property name="minimumSize" stdset="0">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="wholeWords_CKB">
       <property name="text" stdset="0">
        <string>Match Whole Words</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="searchSubFolders_CKB">
       <property name="text" stdset="0">
        <string>Search in Sub Folders</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="regexp_CKB">
       <property name="text" stdset="0">
        <string>Reqular Expression</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="sizeType" stdset="0">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layout_buttons">
     <item>
      <spacer name="horizontalSpacer_31">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="find_PB">
       <property name="text" stdset="0">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_32">
       <property name="sizeType" stdset="0">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>25</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text" stdset="0">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_33">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>find</tabstop>
  <tabstop>findType</tabstop>
  <tabstop>findFolder</tabstop>
  <tabstop>folder_TB</tabstop>
  <tabstop>matchCase_CKB</tabstop>
  <tabstop>wholeWords_CKB</tabstop>
  <tabstop>regexp_CKB</tabstop>
  <tabstop>searchSubFolders_CKB</tabstop>
  <tabstop>find_PB</tabstop>
  <tabstop>cancel_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
